app/components/work_version_detail_component.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class WorkVersionDetailComponent < ApplicationComponent
+  attr_reader :work_version
+
+  def initialize(work_version:)
+    @work_version = work_version
+  end
+
+  def render?
+    return false if current_draft_version?
+
+    i18n_key.present?
+  end
```

Comment:
Is this a Ruby way of writing this? Just curious. I think more typically I would see something like:

```
def render?
  !current_draft_version? && i18n_key.present?
end
```

Obviously what you have works; I am mainly just wondering if the reason you wrote it that way is due to convention. Thanks!

Reply:
@nicktiberi Yeah, that's probably better. This went through a lot of iterations and I wrote it that way so it would look really obvious to me.